src/plugins/intel_npu/thirdparty/CMakeLists.txt

```
@@ -12,3 +12,15 @@ if(ENABLE_ZEROAPI_BACKEND)
     add_library(LevelZero::NPUExt ALIAS level-zero-ext)
     install(TARGETS level-zero-ext EXPORT "${PROJECT_NAME}Targets")
 endif()
+
+#
+# yaml-cpp
+#
+
+if(ENABLE_INTEL_NPU_PROTOPIPE)
+    add_subdirectory(yaml-cpp EXCLUDE_FROM_ALL)
+    # NB: Suppress warnings in yaml-cpp
+    if(NOT MSVC)
```

```suggestion
    if(SUGGEST_OVERRIDE_SUPPORTED)
```
as this option is not supported by all non MSVC compilers.

---

Thanks, fixed in https://github.com/openvinotoolkit/openvino/pull/27087/commits/369948f45b37a5f547a7ae4a6ed2e951a6637098